Algorithm.Python/QuantConnect.Algorithm.Python.csproj

```diff
@@ -140,6 +139,12 @@
     <None Include="TensorFlowNeuralNetworkAlgorithm.py" />
     <None Include="TiingoPriceAlgorithm.py" />
     <None Include="PearsonCorrelationPairsTradingAlphaModelFrameworkAlgorithm.py" />
+    <None Update="UniverseSelectionDefinitionsAlgorithm.py">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Update="ConstituentsQC500GeneratorAlgorithm.py">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
nit - maybe we can go back to existing implementation regarding the imports? seems we didn't use `CopyToOutputDirectory`